packages/config/config/devices/0x010f/fgpb101.json

```diff
@@ -53,179 +91,460 @@
-			"description": "Value SWITCH ON command sent in 3rd association group after a triple click.",
+			"label": "Key Pressed 3 times – value of SWITCH ON command sent to 3rd association group",
+			"description": "1-99 Set Level; 255 Last State",
 			"valueSize": 2,
-			"minValue": 1,
+			"minValue": 99,
 			"maxValue": 255,
 			"defaultValue": 255,
+			"unsigned": true,
 			"readOnly": false,
 			"writeOnly": false,
-			"allowManualEntry": true
+			"allowManualEntry": true,
+			"options": [
+				{
+					"label": "Max Brightness",
+					"value": 99
+				},
+				{
+					"label": "Last State",
+					"value": 255
+				}
+			]
 		},
 		"29": {
-			"label": "Key Held Down – command sent to 3rd group",
-			"description": "Sent associated in 3rd association group after holding the button down.",
+			"label": "Key Held Down – command sent to 3rd association group",
+			"description": "This parameter defines commands sent to devices associated in 3rd association group after holding the button down.\n0- no action\n1- START LEVEL CHANGE UP (brightening)\n2- START LEVEL CHANGE DOWN (dimming)\n3- START LEVEL CHANGE UP/DOWN (brightening/dimming) – alternately",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 3,
 			"defaultValue": 3,
+			"unsigned": true,
 			"readOnly": false,
 			"writeOnly": false,
-			"allowManualEntry": true
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "No Action",
+					"value": 0
+				},
+				{
+					"label": "Dim Level Up (Brightening)",
+					"value": 1
+				},
+				{
+					"label": "Dim Level Down (Dimming)",
+					"value": 2
+				},
+				{
+					"label": "Alternate Brightening / Dimming",
+					"value": 3
+				}
+			]
 		},
 		"30": {
 			"label": "Alarm frame triggers",
-			"description": "Which actions result in sending alarm frames to 4th association group.",
+			"description": "Parameter determines which actions result in sending alarm frames to 4th association group.\n1 - Key Pressed 1 time\n2 - Key Pressed 2 times\n4 - Key Pressed 3 times\n8 - Key Pressed 4 times\n16 - Key Pressed 5 times\n32 - Key Held Down\n64 - Key Released\nNOTE\nValues of parameter 30 may be combined, e.g. 1+2=3 means that alarm frames will be sent after pressing the button once or twice.",
 			"valueSize": 1,
 			"minValue": 1,
-			"maxValue": 127,
+			"maxValue": 64,
 			"defaultValue": 127,
+			"unsigned": true,
 			"readOnly": false,
 			"writeOnly": false,
-			"allowManualEntry": true
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Key Pressed 1 time",
+					"value": 1
+				},
+				{
+					"label": "Key Pressed 2 times",
+					"value": 2
+				},
+				{
+					"label": "Key Pressed 3 times",
+					"value": 4
+				},
+				{
+					"label": "Key Pressed 4 times",
+					"value": 8
+				},
+				{
+					"label": "Key Pressed 5 times",
+					"value": 16
+				},
+				{
+					"label": "Key Held Down",
+					"value": 32
+				},
+				{
+					"label": "Key Released",
+					"value": 64
+				}
+			]
```

Comment:
Bitmask :)

Reply:
@AlCalzone I'm removing this part for now, unless you can tell me how it should look. I haven't delved into the bitmask ones yet.